chromadb/utils/embedding_functions/schemas/bm25_tokenizer.py

```diff
@@ -213,10 +212,8 @@ def stem(self, token: str) -> str:
         return cast(str, self._stemmer.stemWord(token))
 
 
-@lru_cache(maxsize=1)
 def get_english_stemmer() -> SnowballStemmer:
```

Comment:
Can we make this a threadlocal / context ? Creation time might be low but this is a memory-leak over time